### DISCUSSION TITLE
Cannot download install file https://wiki.js.org/install.ps1

### DISCUSSION THREAD

**Discussion by pribylmartinnet** [requarks/wiki#4963] [Error / Bug Report]
Step-by-step: From a PowerShell prompt, run the following command:
https://docs-legacy.requarks.io/wiki/install

There is issue:
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; iex ((New-Object System.Net.WebClient).DownloadString('https://wiki.js.org/install.ps1'))
Exception calling "DownloadString" with "1" argument(s): "Vzdálený server vrátil chybu: (404) Nenalezeno."
At line:1 char:82
+ ... pe]::Tls12; iex ((New-Object System.Net.WebClient).DownloadString('ht ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : WebException

**Comment by Phil-hostwiki** [comment]
Do you have a reason for wanting to install the old Wiki.js V1? 

The project moved domains. The file can be found at https://js.wiki/install.ps1.

**Reply by pribylmartinnet** [reply]
Yes, I would like use MongoDB. Thx
